client/cli/src/params/import_params.rs

```diff
@@ -54,6 +55,15 @@ pub struct ImportParams {
 		default_value = "Interpreted"
 	)]
 	pub wasm_method: WasmExecutionMethod,
+ 
+	/// Enable overriding the on-chain WASM with local wasm on-disk.
+	#[structopt(long)]
+	pub wasm_overwrite: bool, 
+	
+	/// Specify the path where local WASM runtimes are stored.
+	/// These runtimes will override on-chain runtimes when the version matches.
+	#[structopt(long, value_name = "PATH", parse(from_os_str))]
+	pub wasm_overwrite_path: Option<PathBuf>,
```

Comment:
Can we not just merge both? This means if you specify an overwrite path, we enable the overwriting?

Reply:
That was the way I did it at first, but then if we want to have a default path (like base_path/chain/wasm_runtime_overwrites) we would actually need three options I think:
- a way to just enable wasm_overwrites (no path specified)
- a way to enable WASM overwrites with custom path
- a way to have WASM overwrites disabled

If we have just `Option<Path>` we can tell that WasmOverwrites is enabled if it's a `Some`, and then treat it as disabled if it's entirely disabled if as a `None`. However that forces the user to always define a custom path when using overwrites, unless we want to provide a default path if the user enters an empty path or an invalid path, but that seems kindof confusing imo. There might also be some way I missed doing this through `structopt` with only a `Option<>` but after looking through the documentation/discord it does not seem to be apart of the standard they're using

 I don't think there is a good way to decide whether to use the default or specified path, so i created another argument for it.

Reply:
I don't understand why we need a default path? The wasm blobs that will be provided, will be provided manually anyway. I don't see the reason for having a default path for this.